xl/player/player.py

```diff
@@ -397,7 +397,8 @@ def is_playing(self):
             :returns: whether the player is currently playing
             :rtype: bool
         """
-        return self._engine.get_state() == 'playing'
+        return self._engine.get_state() is not None \
```

Comment:
Hm, yes, doesn't look right to me either. There is one case though where it returns `None` that I can reproduce:
1. start exaile with both "Preview Device" and "BPM" disabled
2. open preferences
3. enable "Preview Device"
4. disable "Preview Device"
5. enable "BPM"

Without this change you get a traceback:
```
11:27:25,996:DEBUG   : Provider BPM registered for service mainwindow-info-area-widget with target None (xl.providers)
11:27:25,998:ERROR   : Event callback exception caught! (xl.event)
Traceback (most recent call last):
  File "/path/to/exaile/xl/event.py", line 345, in _emit
    event.data, *cb.args, **cb.kwargs)
  File "/path/to/exaile/xl/providers.py", line 211, in _add_callback
    self.on_provider_added(provider)
  File "/path/to/exaile/xlgui/main.py", line 1129, in on_provider_added
    widget = provider.create_widget(self)
  File "/path/to/exaile/plugins/bpm/__init__.py", line 90, in create_widget
    return BPMWidget(info_area.get_player(), self)
  File "/path/to/exaile/plugins/bpm/__init__.py", line 169, in __init__
    if self.player.is_playing():
  File "/path/to/exaile/xl/player/player.py", line 400, in is_playing
    return self._engine.get_state() == 'playing'
AttributeError: 'NoneType' object has no attribute 'get_state'


Locals at innermost frame:

{ 'self': <xl.player.player.ExailePlayer object at 0x7fc8e2e5eb50>}


11:27:26,  0:DEBUG   : Provider _bpm registered for service track-panel-menu with target None (xl.providers)
11:27:26,  1:DEBUG   : Provider _bpm registered for service playlist-context-menu with target None (xl.providers)
11:27:26,  1:DEBUG   : Loaded plugin bpm (xl.plugins)
```
With this change the traceback is **NOT** gone (and it can't be). I'll revert it.